frontend/src/features/contacts/contact/detail/utils.ts

```diff
@@ -17,6 +19,33 @@ export const sortAddresses = (a1: IContactAddress, a2: IContactAddress) => {
   return -1;
 };
 
+export function getContactInfo(
```

Comment:
this method used to operate on a person or organization, now just operating on the contact methods directly.